velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -927,9 +928,92 @@ void parseFromPattern(
 
 } // namespace
 
+// Evaluates the max size of datetime format result to pre-allocate result
```

Comment:
This comment belongs to .h file.

Returns max size of the formatted string. Can be used to preallocate memory before calling format() to avoid extra copy.